evennia/contrib/rpg/rpsystem/rpsystem.py

```diff
@@ -1539,27 +1397,26 @@ def get_display_name(self, looker, **kwargs):
 
         Keyword Args:
             pose (bool): Include the pose (if available) in the return.
+            ref (str):   Specifies the capitalization for the displayed name.
 
         Returns:
             name (str): A string of the sdesc containing the name of the object,
             if this is defined.
                 including the DBREF if this user is privileged to control
                 said object.
 
-        Notes:
-            The RPObject version doesn't add color to its display.
-
         """
-        idstr = "(#%s)" % self.id if self.access(looker, access_type="control") else ""
+        idstr = "(#%s)" % self.id if self.access(looker, access_type="control") and not kwargs.get("no_id", False) else ""
+        ref = kwargs.get("ref","~")
+    
         if looker == self:
             sdesc = self.key
         else:
             try:
-                recog = looker.recog.get(self)
+                sdesc = looker.get_sdesc(self, ref=ref)
             except AttributeError:
-                recog = None
-            sdesc = recog or (hasattr(self, "sdesc") and self.sdesc.get()) or self.key
-        pose = " %s" % (self.db.pose or "") if kwargs.get("pose", False) else ""
+                sdesc = self.sdesc.get()
+        pose = " %s" % (self.db.pose or "is here.") if kwargs.get("pose", False) else ""
```

Comment:
The `"is here."` string should be lifted out of this code and put in a more easily modifiable location. For example, it can be a property at the class level.